src/test/java/guitests/BoardAutoCreatorTest.java

```diff
@@ -93,4 +99,30 @@ public void workAllocationBoardAutoCreationTest() {
         assertEquals(panelInfos.get(4).getPanelName(), "Work allocated to User 4");
     }
 
+    @Test
+    public void sampleBoardAutoCreationTest() {
+        assertEquals(panelControl.getNumberOfSavedBoards(), 0);
+
+        clickMenu("Boards", "Auto-create", SAMPLE_BOARD);
+
+        PlatformEx.waitOnFxThread();
+        assertNodeExists(hasText(SAMPLE_BOARD + " has been created and loaded.\n\n" +
+                        "It is saved under the name \"" + SAMPLE_BOARD + "\"."));
```

Comment:
Same here.

Also, this can be replaced with `waitUntilNodeAppears`. `Platform.runLater` shouldn't be used like that.